WORKSPACE

```
@@ -27,16 +28,29 @@ load(
     "graknlabs_build_tools",
     "graknlabs_grakn_core",
     "graknlabs_client_java",
-    "graknlabs_protocol"
+    "graknlabs_protocol",
+    "graknlabs_common"
 )
 graknlabs_build_tools()
 graknlabs_grakn_core()
 graknlabs_client_java()
 graknlabs_protocol()
+graknlabs_common()
 
 load("@graknlabs_build_tools//distribution:dependencies.bzl", "graknlabs_bazel_distribution")
 graknlabs_bazel_distribution()
 
+
+##################################
+# Load Distribution dependencies #
+##################################
+
+# needed for @rules_pkg?
+load("@graknlabs_bazel_distribution//github:dependencies.bzl", "tcnksm_ghr")
```

@flyingsilverfin it is needed from `deploy_github` rule (even if your transitive dep uses it). note: you only _loaded_ the macro but haven't invoked it:

```suggestion
load("@graknlabs_bazel_distribution//github:dependencies.bzl", "tcnksm_ghr")
tcnksm_ghr()
```

---

ta